src/cascade/input_data/db/csmr.py

```diff
@@ -92,15 +116,11 @@ def load_csmr_to_t3(execution_context) -> bool:
 
     if _csmr_in_t3(execution_context):
         CODELOG.info(
-            f"csmr data for model_version_id {model_version_id} "
-            f"on '{database}' already exists, doing nothing."
+            f"csmr data for model_version_id {model_version_id} " f"on '{database}' already exists, doing nothing."
```

Comment:
If these are being changed to fit on one line, should the f""'s be combined into one?  Just wondering.